finesse/config.py

```diff
@@ -57,19 +57,9 @@
 DECADES_URL = "http://{host}/decades"
 """The URL of the DECADES sensor data endpoint."""
 
-DECADES_POLL_INTERVAL = 60.0
+DECADES_POLL_INTERVAL = 5.0
 """Poll rate for DECADES sensors."""
 
-DECADES_TOPIC = "decades"
-"""The topic name to use for DECADES sensor-related messages."""
-
-DECADES_QUERY_LIST = [
```

Comment:
Actually, I seem to remember @dc2917 asked Jon about this and he mentioned this specific parameters. Most definitely, he was not interested in all of them (100+ parameters).

Reply:
Ohhh I see. I guess I should have asked :grimacing:.

@jonemurray Is that right? Which parameters do you want to read from the DECADES system?

Reply:
Hmm... maybe what I'll do then is go back to showing just a subset of the readings, but make it user-configurable so that we don't have to redeploy if @jonemurray wants to change this.